src/components/Requirements/RequirementFulfillment.vue

```diff
@@ -1,63 +1,13 @@
 <template>
   <div class="subrequirement">
```

Comment:
nit: should we rename this class because it's technically for the requirementfulfillment component now and subrequirements are the slots?

Reply:
Turns out there is no css class for `subrequirement`, so i will just remove it